code/modules/paperwork/photocopier.dm

```diff
@@ -351,10 +487,14 @@
 	else
 		to_chat(user, "<span class='notice'>[src]'s laser printing mechanism is already overloaded!</span>")
 
+
+//TODO: Add an emp_act effect for photocopiers -sirryan
```

Comment:
TODO 👀

Reply:
yessir, I might do this before merging but it's not high on my priority list so I may save it for another PR